docs/sphinx/dev_guide/ci.rst

```diff
@@ -17,14 +17,17 @@ Continuous Integration (CI) Testing
                  viewed by clicking the appropriate link in the **checks** 
                  section of a GitHub pull request.
 
-The RAJA project uses two CI tools to run tests:
-
-  * **Azure Pipelines** runs builds and tests for Linux, Windows, and MacOS 
-    environments using compilers in container images maintained in the
-    `RSE Ops Project <https://github.com/rse-ops/docker-images>`_.
-    While we do some GPU builds on Azure, RAJA tests are only run for CPU-only 
-    builds. The current set of builds run on Azure can be seen by looking at
-    the ``RAJA/azure-pipelines.yml`` and ``RAJA/Dockerfile`` files.
+The RAJA project uses threes CI tools to run tests:
```

Comment:
Should this be "uses three CI tools" instead of "threes"?

Reply:
Good catch. Fixed.